torchvision/transforms.py

```diff
@@ -236,13 +236,52 @@ class Pad(object):
         padding (int or sequence): Padding on each border. If a sequence of
             length 4, it is used to pad left, top, right and bottom borders respectively.
         fill: Pixel fill value. Default is 0.
+        type:padding type: constant,reflect,edge,symmetric
     """
 
-    def __init__(self, padding, fill=0):
+    def __init__(self, padding, fill=0,type="constant"):
```

Comment:
nit: I would move `fill` after `type` as `fill` is only useful for the `constant` type.